### PR TITLE
Fix deprecated alias_method_chain 

### PR DESCRIPTION
**Pull request by chipset508**
The reason we need to bump version here is after [this PR](https://github.com/Thinkei/xero_gateway/pull/36) the author did not bump gem version

Change:
- Fix bug alias_method_chain

### CODE REVIEW

**Review by keymastervn — APPROVED** [review]
LGTM

**Review by anlnt — DISMISSED** [review]
🥥

**Review by vutienduongeh — APPROVED** [review]
go

**Review by Hoangtran138 — APPROVED** [review]

File: lib/oauth/oauth_consumer.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def http_with_ssl_client_certificates(*args)
       end
     end
 
-    alias_method :ssl_client_certificates, :http
-    alias_method :http, :ssl_client_certificates
+    alias_method :http_without_ssl_client_certificates, :http
```
Comment (phamtuan2993) [inline lib/oauth/oauth_consumer.rb:10]: 👀  cái này là cái gì zậy?

**Review by phamtuan2993 — DISMISSED** [review]

**Review by phamtuan2993 — APPROVED** [review]

**Comment by phamtuan2993** [comment]
Cc a @vutienduongeh approve lại phát a ơi :awthanks: ms fix cái ci

**Review by vutienduongeh — APPROVED** [review]

**Comment by keymastervn** [comment]
Hi folks, FYI I am migrating the CI config to another PR, then I'll undo the last 2 commits so that we can have `development` for pre-release and testing purpose, `master` for the post-release major bump.

**Review by phamtuan2993 — APPROVED** [review]
xịn thật sự

File: app.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xero_gateway",
-  "version": "3.2.23",
+  "version": "3.2.23.alpha",
```
Comment (phamtuan2993) [inline app.json:3]: ủa khoan, cái này là staging :okay:

**Pull request merged**